docs/data-access/daco/renew-and-close.md

```diff
@@ -5,52 +5,63 @@ sidebar_label: Attesting, Renewing, & Closing
+| ------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Attestation Required**  | The application requires an annual agreement in order to continue with the last year of access. If this is not completed by the annual date, access will be paused until you attest. <br/> <br/> ![DACO application requires annual attestation](/assets/data-access/daco-annual-attestation.png)                 | In this stage, you can still download the DACO signed PDF using the **Approved PDF** top button. <br /><br /> ![Download button for DACO approved PDF](/assets/data-access/download-approved.png)                                 |
+| **Expired**               | Access will expire 2 years after the date of approval. You will have up to 90 days after the expiry date to renew this application for an additional 2 years of access. <br/> <br/> ![DACO application in Expired state](/assets/data-access/daco-expired-renewable.png)                                          | In this stage, you can still download the DACO signed PDF using the **Approved PDF** top button. <br /><br /> ![Download button for DACO approved PDF](/assets/data-access/download-approved.png)                                 |
+| **Expired Non-renewable** | If you do not renew the application within 90 days after expiry, the application will be closed permanently. <br/> <br/> ![DACO application in Expired Non-renewable state](/assets/data-access/daco-expired-nonrenewable.png)                                                                                    | You can download the DACO signed PDF using the **Approved PDF** top button. <br /><br /> ![Download button for DACO approved PDF](/assets/data-access/download-approved.png)                                                      |
+| **Renewal Application**   | Upon choosing to renew an application, a new draft application is generated, linked to the original application. This new draft must be completed for renewal. The original application cannot be edited. <br/> <br/> ![Renewal DACO application](/assets/data-access/daco-renewal.png)                           | A **Draft PDF** of the new application can be downloaded, which will display a watermark, useful for collaboration on the renewal process. <br /><br /> ![Download button for draft PDF](/assets/data-access/download-draft.png)  |
```

Comment:
there are a couple editing options available on the original application, as long as it is still in `APPROVED` state - collaborators can be added/removed, and ethics docs can be added. However any changes to the original after the renewal draft is created would not be transferred to the renewal.
tldr; not sure if we should include that `The original application cannot be edited` line?